postgresql/restore.sh

```diff
@@ -0,0 +1,18 @@
+
+#!/bin/bash
+
+# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
+#          sudo yum -y install postgresql96
+
+# Before running the restore, you'll have to stop the ECS task to close any open connections.
+# Then, run the following to drop the database.
+# psql -U concordia -h $POSTGRES_HOST
+# \c postgres
+# drop database concordia;
+
+export POSTGRES_HOST=
+export DUMP_FILE=concordia.dmp
+
+pg_restore --create --clean -U concordia -h $POSTGRES_HOST -Fc --dbname=postgres --no-owner --no-acl $DUMP_FILE
```

Comment:
This may be version specific but with 10.5 this command failed and told me to use `psql` instead for a plain text file.

Reply:
Yeah this command is for a dump created with the -Fc flag.

Reply:
Oh, that's not the dump I pulled from S3?

Reply:
No, I used the one created by the command in `dump.sh` after my attempt to use the plain text SQL to reload failed.